docs/source/enable/constraints_layout.rst

```diff
@@ -90,43 +90,54 @@ also available in Enable. The layout helpers are:
 
 :data:`spacer`: Creates space between two adjacent components.
 
-.. function:: horizontal(*components[, spacing=10])
+.. function:: hbox(*components[, spacing=10, margins=...])
```

Comment:
Why `...` instead of 0?

Reply:
Because `margins` has several possible values which are enumerated below this line

Reply:
Yes, I thought this line was meant to indicate the default value that the parameter takes on, not to indicate the range of types that it accepts, which is why I thought it might read 0 instead.

It doesn't matter too much; I was just curious since I hadn't seen ellipsis used like this.